thanos/charts/templates/store-svc.yaml

```
@@ -1,6 +1,7 @@
 # SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
 # SPDX-License-Identifier: Apache-2.0
 
+{{ if .Values.thanos.compactor.enabled }}
```

```suggestion
{{ if .Values.thanos.store.enabled }}
```

suppose this should be triggered with store?